src/applications/vaos/services/appointment/index.js

```diff
@@ -0,0 +1,28 @@
+import { getConfirmedAppointments as getConfirmedVistaAppointments } from '../../api';
```

Comment:
Why did you alias the name here?

Reply:
I was originally going to name the service function `getConfirmedAppointments` which would conflict with the API function name so I aliased it, but ended up going with `getBookedAppointments` and forgot to remove the alias.  Removed.